addons/sale_project/models/product_template.py

```diff
@@ -37,6 +37,10 @@ def _selection_service_policy(self):
         'project.project', 'Project Template', company_dependent=True, copy=True,
         domain='[("is_template", "=", True)]',
     )
+    task_template_id = fields.Many2one('project.task', 'Task Template',
+        domain="[('is_template', '=', True), ('project_id', '=', project_id)]",
```

Comment:
did you recently change the domain? I thought we also take the `project_template_id` into account.

Reply:
> did you recently change the domain? I thought we also take the `project_template_id` into account.

When we create a product as 'Create on Order → Task', the task template field is displayed. Therefore, I used only the project_id in the domain. Is there any reason we need to use the project template in the domain?

I shared a small video in the task chatter."

Reply:
ok thanks.